testcases/OpTestInbandIPMI.py

```diff
@@ -152,13 +73,25 @@ def run_ipmi_cmd_on_host(self, i_cmd):
             raise OpTestError(l_msg)
```

Comment:
The above code fails due to %c for character instead of string and global OpTestError is not available.
Can you replace this with below line.
self.assertFalse(int(l_res[-1]), "IPMI: command %s failed" % l_cmd)